lesson-10/jquery.js

```diff
@@ -0,0 +1,27 @@
+const express =require("express");
+const fs =require("fs");
+const gm =require("gm");
+const bodyparser =require("body-parser");
+const app =express();
+app.listen(3610);
+app.use(express.static("public"));
+app.use(bodyparser.json({limit: '500mb'}));
+
+  app.post('/submit',(req,res)=>{
+    fs.appendFile('/home/madhub/garagescript/lesson-10/public/notes.txt', `Name: ${req.body.name} Comment: ${req.body.comment}`);
+  });
+
+  app.post('/pics', (req,res)=>{
+    let picdata =req.body.canvas.replace('data:image/png;base64,', '');
+    let picpath =`/home/madhub/garagescript/lesson-10/public/pics/${req.body.name}.png`;
+    let memePath =`/home/madhub/garagescript/lesson-10/public/meme/${req.body.name}.png`;
```

Comment:
I do not think you need a different path for meme, you can just override the picture cuz you are not using the picture without meme anyways.